source/codegen/service_helpers.py

```diff
@@ -1,33 +1,4 @@
 import common_helpers
-import proto_helpers
-
-RESERVED_WORDS = [
-    'abstract', 'as',
-    'base', 'bool', 'break','byte',
-    'case', 'catch', 'char', 'checked', 'class', 'const', 'continue',
-    'decimal', 'default', 'delegate', 'do', 'double',
-    'else', 'enum', 'event', 'explicit', 'extern',
-    'false', 'finally', 'fixed', 'float', 'for', 'foreach',
-    'goto',
-    'if', 'implicit', 'in', 'int', 'interface', 'internal', 'is',
-    'lock', 'long',
-    'namespace', 'new', 'null',
-    'object', 'operator', 'out', 'override',
-    'params', 'private', 'protected', 'public',
-    'readonly', 'ref', 'return',
-    'sbyte', 'sealed', 'short', 'sizeof', 'stackalloc', 'static', 'status', 'string', 'struct', 'switch',
-    'this', 'throw', 'true', 'try', 'typeof',
-    'uint', 'ulong', 'unchecked', 'unsafe', 'ushort', 'using',
-    'virtual', 'void', 'volatile',
-    'while'
-]
-
-def sanitize_names(parameters):
```

Comment:
Note: `sanitize_names` and the `RESERVED_WORDS` array were moved to `metadata_mutation.py`.